src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -115,6 +115,14 @@ trait AdminClient {
     options: Option[DescribeConfigsOptions] = None
   ): Task[Map[ConfigResource, KafkaConfig]]
 
+  /**
+   * Get the configuration for the specified resources async.
```

Comment:
```suggestion
   * Gives the configuration for the specified resources.
```
There is no need to state that the response is asynchronously.

Reply:
then the description will be the same as for the existing `describeConfigs` method, that I wanted to avoid